### PR TITLE
fix xpath working with stream

### PR DESCRIPTION
**Pull request by nsolov**

### CODE REVIEW

File: src/cls/ZPM/Package.cls [hunk 1]
```diff
@@ -286,16 +286,30 @@ ClassMethod GetModuleMeta(url As %String) As %DynamicObject
...
   }
   Set ht.Location = tComponents("path")_"master/module.xml"
-    
   $$$ThrowOnError(ht.Get())
   If (ht.HttpResponse.StatusCode'=200) {
     $$$ThrowStatus($$$ERROR(5001,"Unable to download XML "_ht.Location))
   }
-  $$$ThrowOnError(##class(%XML.XPATH.Document).CreateFromStream(ht.HttpResponse.Data, .xpathdoc))
+  Set tmpFolder = $$$FileTempDir
+  Do ##class(%File).CreateDirectory(tmpFolder)
```
Comment (daimor) [inline src/cls/ZPM/Package.cls:294]:
```suggestion
```
$$$FileTempDir returns already created folder

File: src/cls/ZPM/Package.cls [hunk 2]
```diff
@@ -286,16 +286,30 @@ ClassMethod GetModuleMeta(url As %String) As %DynamicObject
...
+
+  Set stream=##class(%Stream.FileCharacter).%New()
+  Do stream.LinkToFile(tmpFolder_"/module.xml")
+  Set stream.TranslateTable = "UTF8"
+  Do stream.CopyFrom(ht.HttpResponse.Data)
+  Do stream.%Save()
+
+  set binaryStream=##class(%Stream.FileBinary).%New()
+  do binaryStream.LinkToFile(tmpFolder_"/module.xml")
```
Comment (daimor) [inline src/cls/ZPM/Package.cls:303]: binaryStream or stream?

File: src/cls/ZPM/Package.cls (hunk 1 above)
Comment (nsolov) [reply src/cls/ZPM/Package.cls]: deleted in 2 places in this file

File: src/cls/ZPM/Package.cls (hunk 2 above)
Comment (nsolov) [reply src/cls/ZPM/Package.cls:301]: binaryStream 
CreateFromStream method requires it. "Use this method to create an instance of an %XML.XPATH.Document from a binary stream. It's important to use a binary stream ..." https://docs.intersystems.com/irislatest/csp/documatic/%25CSP.Documatic.cls?PAGE=CLASS&LIBRARY=%25SYS&CLASSNAME=%25XML.XPATH.Document

**Pull request merged**